_posts/2015-08-10-type-driven-development.html

```diff
@@ -44,7 +44,7 @@
     My problem was that I didn't know how long it took to handle a message, so the Polling Consumer would have to measure that as well, constantly updating its estimates of how long it takes to handle a message.
   </p>
   <p>
-    This wasn't a harder problem that I could <a href="http://blog.ploeh.dk/2014/09/25/faking-a-continuously-polling-consumer-with-scheduled-tasks">originally solve it in a highly coupled imperative fashion</a>. I wasn't happy with that implementation, though, so I'm happy that there's a much more <em>incremental</em> approach to the problem.
+    This wasn't a sufficiently hard problem to thwart my <a href="http://blog.ploeh.dk/2014/09/25/faking-a-continuously-polling-consumer-with-scheduled-tasks">originally solving it in a highly coupled imperative fashion</a>. I wasn't happy with that implementation, though, so I'm happy that there's a much more <em>incremental</em> approach to the problem.
```

Comment:
FWIW "I could originally solve" is kind of a strange combination of words regardless of the rest of the sentence. That exact phrase has no google hits other than this blog post, and even "could originally solve" is only used in sentences with the sense of somethings abilities changing over time rather than a choosing to do something differently.

Not quite sure why I'm writing a long paragraph on exact choices of words in a blog post, I just happened to click the "open pull requests" button.

FWIW I'd suggest "This is a sufficiently easy problem that I originally used a highly coupled, imperative solution", just because it's shorter and avoids the maybe-double-negative "wasn't sufficiently hard to prevent", but then again colorless green ideas sleep furiously.